.codeclimate.yml

```diff
@@ -1,6 +1,6 @@
 ---
-engines:
-
+version: 2
+plugins:
   duplication:
```

Comment:
`duplication` is one of the things that is now part of the stock checks in version 2, so you should be able to remove this.